scout/server/blueprints/variants/controllers.py

```diff
@@ -1072,20 +1072,23 @@ def check_form_gene_symbols(
     clinical_symbols = store.clinical_symbols(case_obj)
 
     for hgnc_symbol in hgnc_symbols:
-        hgnc_gene = store.hgnc_genes_find_one(hgnc_symbol, genome_build)
+        # Retrieve a gene with "hgnc_symbol" as hgnc symbol or a list of genes where hgnc_symbol is among the aliases
+        hgnc_genes = list(store.gene_by_symbol_or_aliases(symbol=hgnc_symbol, build=genome_build))
 
-        if hgnc_gene is None:
+        if not hgnc_genes:
             not_found_symbols.append(hgnc_symbol)
-        elif is_clinical is False:  # research variants
-            updated_hgnc_symbols.append(hgnc_symbol)
-        elif hgnc_gene["hgnc_id"] in clinical_hgnc_ids:  # clinical variants
-            updated_hgnc_symbols.append(hgnc_symbol)
-            if hgnc_symbol not in clinical_symbols:
-                # clinical symbols from gene panels might not be up to date with latest gene names
-                # but their HGNC id would still match
-                outdated_symbols.append(hgnc_symbol)
-        else:  # clinical variants
-            non_clinical_symbols.append(hgnc_symbol)
+
+        for hgnc_gene in hgnc_genes:
+            if is_clinical is False:  # research variants
+                updated_hgnc_symbols.append(hgnc_symbol)
+            elif hgnc_gene["hgnc_id"] in clinical_hgnc_ids:  # clinical variants
+                updated_hgnc_symbols.append(hgnc_symbol)
```

Comment:
```suggestion
                updated_hgnc_symbols.append(hgnc_gene["hgnc_symbol"])
```
That way the updated symbol goes in.